RELEASE_GUIDELINE.md

```diff
@@ -0,0 +1,40 @@
+# RELEASE GUIDELINES
+- There is a scheduled release on every 15th of the month on the following repositories:
+  - [Litmus](https://github.com/litmuschaos/litmus)
+  - [Chaos-Operator](https://github.com/litmuschaos/chaos-operator)
+  - [Chaos-Exporter](https://github.com/litmuschaos/chaos-exporter)
+
+- These repositories use release version according to the [Semantic Versioning](https://semver.org/)
+- These repositories create a new docker image tag after the release is done. Following are the docker images:
+  - [Litmus Ansible Runner](https://cloud.docker.com/u/litmuschaos/repository/docker/litmuschaos/ansible-runner)
+  - [Chaos-Operator](https://cloud.docker.com/u/litmuschaos/repository/docker/litmuschaos/chaos-operator)
+  - [Chaos-Exporter](https://cloud.docker.com/u/litmuschaos/repository/docker/litmuschaos/chaos-exporter)
+- ### Releases
```

Comment:
Before releases, let us have a section explaining flow of a release, i.e.,

- Sprint planning/, Feature freeze
- Feature development
- Code freeze, RC1
- Test Phase, Bug-fix, Branch Creation
- Cherry Picks
- Release

Reply:
May also be good to mention the role of the kanban board and have milestone attached to the issues along with project